scipy/stats/tests/test_stats.py

```diff
@@ -5540,6 +5542,30 @@ def test_2d_matrix_axis1(self):
         desired = matrix([[19.2, 63.03939962, 103.80078637]]).T
         check_equal_hmean(matrix(a), desired, axis=1)
 
+    def test_weights_1d_list(self):
```

Comment:
We should also test in more than 1D since we have axis.

Reply:
Test updated with 2D array.

Reply:
Please add a check on another axis as well (transpose the current test for instance)